modules/core/src/main/java/org/apache/ignite/dump/DumpReader.java

```diff
@@ -89,13 +90,13 @@ public DumpReader(DumpReaderConfiguration cfg, IgniteLogger log) {
 
                 Map<Integer, List<String>> grpToNodes = new HashMap<>();
 
-                Set<Integer> cacheGroupIds = cfg.cacheGroupNames() != null 
+                Set<Integer> cacheGrpIds = cfg.cacheGroupNames() != null
```

Comment:
Seems these changes not related to the ticket. Please, revert.

Reply:
Reverted